app/views/layouts/_header.html.slim

```diff
@@ -1,20 +1,22 @@
 div class="bg-black"
   div class="row"
     div class="title-bar" data-responsive-toggle="main-menu" data-hide-for="medium"
-      button class="menu-icon" type="button" data-toggle=""
+      button[class="menu-icon" type="button" data-toggle]
       div class="title-bar-title" Menu
 
     div class="top-bar" id="main-menu"
       div class="top-bar-left"
-        ul class="dropdown menu" data-dropdown-menu=""
-          li class="menu-text" Pangeas
-          li: a href='#' Início
-          li: a href='#' Eventos & Pesquisa
-          li: a href='#' Vida em Equilíbrio
-          li: a href='#' Cursos Pangeas
+        ul[class="dropdown menu" data-dropdown-menu]
+          li class="menu-text"
+            = image_tag "logo-pangeas.png"
+          li class="menu-item"
+            = link_to "Início", root_path
+          / li class="menu-item": a href='#' Eventos & Pesquisa
+          / li class="menu-item": a href='#' Vida em Equilíbrio
+          / li class="menu-item": a href='#' Cursos Pangeas
       div class="top-bar-right"
```

Comment:
You can use just `.top-bar-right` (divs are considered a default tag in Slim, so you don't even need to use the tag :D)